prac_07/programming_language.py

```diff
@@ -0,0 +1,46 @@
+# Adrianto Hartono
+# 14347930
+
+"""
+CP1404/CP5632 Practical - Suggested Solution
+Programming Language class with tests.
+"""
+
+
+class ProgrammingLanguage:
+    """Represent information about a programming language."""
+
+    def __init__(self, name, typing, reflection, year, pointer_arithmetic):
+        """Construct a ProgrammingLanguage from the given values."""
+        self.name = name
+        self.typing = typing
+        self.reflection = reflection
+        self.year = year
+        self.pointer_arithmetic = pointer_arithmetic
+
+    def __repr__(self):
+        """Return string representation of a ProgrammingLanguage."""
+        return f"{self.name}, {self.typing} Typing, Reflection={self.reflection}, First appeared in {self.year}, Pointer Arithmetic={self.pointer_arithmetic}"
+
+    def is_dynamic(self):
+        """Determine if language is dynamically typed."""
+        return self.typing == "Dynamic"
+
+
```

Comment:
I think it would be better to either delete the run test function or comment out the function